src/index.js

```diff
@@ -13,12 +13,18 @@ Gun.on('opt').event((gun, options) => {
 	const adapter = Adapter.from(level);
 
 	const { wire } = gun.__.opt;
+	var wireGet = wire.get
```

Comment:
Lower priority: for consistency, can we change `var` to `const` and add semicolons? `npm run lint` should help.